EIPS/eip-5252.md

```diff
@@ -0,0 +1,57 @@
+
+## Motivation
+
+Finance in the crypto ecosystem is facing a huge trust issue. Smart contracts are often proxies, with the actual logic of the contract hidden away in a separate logic contract. Many projects include a multi-signature "wallet" with unnecessarily-powerful permissions. And it is not possible to independently verify that stablecoins have enough real-world assets to continue maintaining their peg, creating a large loss of funds (such as happened in the official bankruptcy announcement of Celsius and UST de-pegging and anchor protocol failure). One should not trust exchanges or other third parties with one's own investments.
+
+The pattern empowers more self-sovereignty and gives more credentials(obviously with an Account-bound token) than locking financial data in the operating team's contract.
+
+Segregation between investor's fund and operation fee is clearly specified in the smart contract, so investors can ensure safety from arbitrary loss of funds by the operating team's control.
+
+## Specification
+
+The pattern consists of 5 components, manager, factory, finance, account-bound token, and extension.
+
+### Diagram
+
+<figure>
+<img src="../assets/eip-5252/diagram.svg" alt="Trulli" style="width:100%">
+<figcaption align = "center"><b>Fig 1 - Component Diagram of EIP-5252</b></figcaption>
+</figure>
+
+
+**`Manager`**: **`Manager`** contract acts as an entry point to interact with the investor. The contract also stores parameters for **`Finance`** contract.
+
+**`Factory`**: **`Factory`** contract manages contract bytecode to create for managing investor's fund and clones **`Finance`** contract on **`Manager`** contract's approval. It also mints account-bound tokens to interact with the `Finance` contract.
+
+**`Finance`**: **`Finance`** contract specifies all rules on managing an investor's fund. The contract is only accessible with an account that has an Account-bound token. When an investor deposits a fund to **`Manager`** contract, the contract sends the fund to **`Finance`** contract account after separating fees for operation. 
+
+**`Account-bound token`**: **`Account-bound token`** contract in this EIP can bring the **`Finance`** contract's data and add metadata. For example, if there is a money market lending **`Finance`** contract, its **`Account-bound token`** can show how much balance is in agreement using SVG.
+
+**`Extension`**: **`Extension`** contract is another contract that can utilize locked funds in **`Finance`** contract. The contract can access with **`Finance`** contract on operator's approval managed in **`Manager`** contract.
+
+## Backwards Compatibility
+This EIP has no known backward compatibility issues.
+
+## Reference Implementation
+
+[Reference implementation](https://github.com/digitalnativeinc/standard-evm/tree/master/contracts/vaults/meter) is a CDP stablecoin lending project.
```

Comment:
No external links.  You need to either inline the code or add it to `../assets/eip-5252/` or remove this section entirely (it is optional).  Note: Code included should be as small/slim as possible, please do not include an entire build system/project, just enough code to exemplify how one might implement this standard.  Pseudocode or incomplete code is acceptable

Reply:
ok I am making example contracts.